docs/generate-tables.sh

```diff
@@ -1,31 +1,30 @@
 #!/bin/bash
 # Generate the documentation tables as docs/generated/OJP.html from the .xsd schema files
```

Comment:
The path `docs/generated/OJP.html` should be changed to ojp.html at least and if `contab` is not removed (which I would favor), has to be added